src/hb-ot-post-table.hh

```diff
@@ -82,6 +82,34 @@ struct post
     return_trace (true);
   }
 
+  inline bool subset (hb_subset_plan_t *plan) const
+  {
+    hb_blob_t *post_blob = OT::Sanitizer<post>().sanitize (hb_face_reference_table (plan->source, HB_OT_TAG_post));
+    post *post_table = (post *) hb_blob_get_data (post_blob, nullptr);
+
+    post *post_prime = (post *) malloc (post::static_size);
```

Comment:
Please allocate fixed-size objects using calloc(..., 1).

Reply:
Done. Reworked to use create sub blob.